brave/brave-core
# Show import instruction for empty bookmark

Resolves https://github.com/brave/brave-browser/issues/3568

![image](https://user-images.githubusercontent.com/6786187/82666519-0fe31800-9c71-11ea-9490-2ee0f1eafc72.png)


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Launch Browser with fresh profile
2. Check bookmark bar is empty and import instruction is visible
3. Bookmark any pages and check import instruction is hidden
4. Delete all bookmarks and check import instruction is visible again

`npm run test brave_browser_tests -- --filter=BookmarkBarTest*`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
